frontend/src/utils/notifications.ts

```diff
@@ -22,6 +22,27 @@ export const contactAdministrator = (
   display(message, { variant: variant });
 };
 
+/**
+ * Display an info alert, followed by an invitation to report an issue.
+ *
+ * Contrary to `contactAdministrator` this function always use the info alert
+ * variant to notify the user of the presence of non-impacting error. It should
+ * not be used to report blocking errors or errors requiring an action from the
+ * user.
+ */
+export const contactAdministratorLowSeverity = (
+  display: ProviderContext['enqueueSnackbar'],
+  message?: string
+) => {
+  if (!message) {
+    message = 'Oops, a non impacting error occurred.';
+  }
+  display('Please, contact an administrator to report the issue.', {
+    variant: 'warning',
+  });
+  display(message, { variant: 'info' });
```

Comment:
This is really minor, but I find a bit disturbing to trigger 2 messages with different severity levels. As the error is not really severe, could "info" be used instead of "warning" here?

Reply:
Of course :+1:

Reply:
Fixed by d63b49c10f9bb4ef67522571e7ed04d231069a66.